dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/JSF23WebSocketTests.java

```diff
@@ -151,7 +156,13 @@ public void testOpenAndCloseWebsocket() throws Exception {
 
             // Construct the URL for the test
             String contextRoot = "WebSocket";
-            URL url = JSFUtils.createHttpUrl(jsf23CDIWSOCServer, contextRoot, "OpenCloseWebSocketTest.jsf");
+
+            URL url;
+            if (JakartaEE10Action.isActive()) {
+                url = JSFUtils.createHttpUrl(jsf23CDIWSOCServer, contextRoot, "faces40/OpenCloseWebSocketTest.jsf");
+            } else {
+                url = JSFUtils.createHttpUrl(jsf23CDIWSOCServer, contextRoot, "jsf23/OpenCloseWebSocketTest.jsf");
+            }
```

Comment:
These wsoc test are commented out in the jsf 2.3 fat bucket, but I'm curious to see if they work here. We can comment them out if they fail here too.

Reply:
Yes, they did pass. Not sure why the same tests in other FAT  failed in the builds, but so far it looks good here. RC2 updated some websocket code, so it could be related